v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FConstants.scala

```diff
@@ -0,0 +1,244 @@
+package org.broadinstitute.monster.etl.v2f
+
+/**
+  * constants for v2f that allow for transforms of the converted tsv to json
+  */
+sealed trait V2FConstants {
+
+  /**
+    * the name of the tsv table that is being converted to Json and transformed
+    */
+  def tableName: String
+
+  /**
+    * relative path to the tsv that will be converted to Json and transformed
+    */
+  def tsvPath: String
```

Comment:
Would rename the var itself, too:
```suggestion
  def tsvPattern: String
```